eventfiles/models.py

```diff
@@ -0,0 +1,29 @@
+import os
+from django.db import models
+from events.models import EventParticipant
+from django.core.exceptions import ValidationError
+
+
+def upload_to_function(instance, filename):
+    return os.path.join('files/%s/' % instance.participant_id.event_id.id, filename.split("/")[-1])
+
+
+class EventFile(models.Model):
+    participant_id = models.ForeignKey(EventParticipant, on_delete=models.RESTRICT)
+    file = models.FileField(upload_to=upload_to_function)
+
+    @staticmethod
+    def validate_unique_file(up_file, up_participant_id):
+        if EventFile.objects.filter(file=up_file, participant_id=up_participant_id):
+            raise ValidationError('that file already exist in meeting')
+
+    def __str__(self):
+        return str(self.file.name)
```

Comment:
In my opinion this is a more readable and informative way of displaying the event file instance.

```suggestion
    def __str__(self):
        _, _, file_name = self.file.name.split("/")
        return f"{self.participant_id} - {file_name}"
```

Reply:
3rd option - something like:
```suggestion
    def __str__(self):
        return f"{self.participant_id} - {os.path.basename(self.file.name)}"
```